packages/site/cms-site/src/iframebridge/IFrameBridge.tsx

```diff
@@ -112,6 +118,7 @@ export const IFrameBridgeProvider: React.FunctionComponent = ({ children }) => {
                     sendMessage({ cometType: IFrameMessageType.HoverComponent, data: { route } });
                 },
                 sendMessage,
+                contentScope,
```

Comment:
The current contentScope could also be put in a ContentScopeContext that is also used when rendering a site the "standard" way and site preview. So any component can use some kind of useScope.

But that is out of scope for this PR, and might even be implemented app side only. (then we should have it in demo touogh)

And: Do you think contentScope should be required? (i.e. don't render when content scope is not yet received)

Reply:
Do you think contentScope should be required? (i.e. don't render when content scope is not yet received)